components/shared/richText/RichTextElement.tsx

```diff
@@ -1,19 +1,10 @@
 import { ArrowTopRightOnSquareIcon } from "@heroicons/react/24/solid";
 import { Elements } from "@kontent-ai/delivery-sdk";
-import {
-  IPortableTextComponent,
-  IPortableTextImage,
-  IPortableTextInternalLink,
-  IPortableTextItem,
-  IPortableTextTable,
-  nodeParse,
-  transformToPortableText,
-} from "@kontent-ai/rich-text-resolver";
+import { PortableTextItem, transformToPortableText } from "@kontent-ai/rich-text-resolver";
+import { PortableTextReactResolvers } from '@kontent-ai/rich-text-resolver/utils/react';

```

Comment:
get rid of `<PortableText>` import from the external package and import it directly from the react utils. it's almost the same, but contains the default resolution for things like `sub` and `sup` we removed in this PR.